WooCommerce/Classes/POS/Presentation/CardReaderConnection/CardReaderConnectionStatusView.swift

```diff
@@ -111,7 +111,7 @@ private extension CardReaderConnectionStatusView {
         static let imageDimension: CGFloat = 12
         static let progressIndicatorDimension: CGFloat = 10
         static let progressIndicatorLineWidth: CGFloat = 2
-        static let font = POSFontStyle.posDetailEmphasized
+        static let font = POSFontStyle.posBodyMediumRegular
```

Comment:
Just double checking: This font is not only renamed to its `posBodySmallEmphasized` equivalent, but also changed from 16 semibold to 20 semibold (which would be 20 bold if @staskus comment on  https://github.com/woocommerce/woocommerce-ios/pull/15114/files#r1950433532 is applied.

Reply:
Some of the updates including this one is from looking up the use case in the design. In this case, the text style is bodyMediumRegular as in 1qcjzXitBHU7xPnpCOWnNM-fi-436_4477. That said, this component hasn't been finalized and is subject to change in https://github.com/woocommerce/woocommerce-ios/pull/15103.